packages/swarm-docs/src/components/nav.js

```diff
@@ -6,11 +6,18 @@ type Props = {
     fileNames: Array<string>
 };
 
-const Nav = (props: Props): React.Element<'div'> => {
```

Comment:
Why remove this? You can probably add it back

Reply:
vscode was giving me a warning on it. But happy to put it back.